specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/event-grid-schema/incoming_call.json

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "e80026e7-e298-46ba-bc42-dab0eda92581",
+  "topic": "/subscriptions/{subscription-id}/resourceGroups/{group-name}/providers/Microsoft.Communication/communicationServices/{communication-services-resource-name}",
+  "subject": "/caller/{caller-id}/recipient/{recipient-id}",
+  "data": {
+    "to": {
+      "kind": "communicationUser",
+      "rawId": "{recipient-id}",
+      "communicationUser": {
+        "id": "{recipient-id}"
+      }
+    },
+    "from": {
+      "kind": "communicationUser",
+      "rawId": "{caller-id}",
+      "communicationUser": {
+        "id": "{caller-id}"
+      }
+    },
+    "serverCallId": "{server-call-id}",
+    "callerDisplayName": "VOIP Caller",
+    "customContext": {
+      "sipHeaders": {
```

Comment:
What about customHeaders?

Reply:
Its is optional it can be available or not available.

Reply:
It would be helpful to include it, especially since there is a question about whether it should be a dictionary.